sql/core/src/test/scala/org/apache/spark/sql/sources/DataSourceAnalysisSuite.scala

```diff
@@ -35,6 +35,7 @@ class DataSourceAnalysisSuite extends SparkFunSuite with BeforeAndAfterAll {
 
   override def beforeAll(): Unit = {
     super.beforeAll()
+    SQLConf.get.unsetConf(SQLConf.CASE_SENSITIVE)
```

Comment:
we should do it in `aferAll`, to avoid this test suite changing the global conf object.